src/modules/payment-module/sablier-lockup/interfaces/IStreamManager.sol

```diff
@@ -1,11 +1,9 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 pragma solidity >=0.8.22;
 
-import { Broker, Lockup } from "@sablier/lockup/src/types/DataTypes.sol";
+import { Lockup } from "@sablier/lockup/src/types/DataTypes.sol";
```

Comment:
We recommend importing `Lockup` from `@sablier/lockup/src/types/Lockup.sol`. There are not different libraries for different contract types.